source/conf.py

```diff
@@ -40,6 +42,8 @@
         "rapids_notebooks_container": f"rapidsai/notebooks:{nightly_version + 'a'}-cuda12.5-py3.12",
         "rapids_conda_channels": "-c rapidsai-nightly -c conda-forge -c nvidia",
         "rapids_conda_packages": f"rapids={nightly_version} python=3.12 cuda-version=12.5",
+        "rapids_pip_index": "https://pypi.anaconda.org/rapidsai-wheels-nightly/simple",
+        "rapids_pip_version": f"{nightly_version}.0a0,<={nightly_version}",
```

Comment:
```suggestion
        "rapids_pip_version": f"{nightly_version}.*,>=0.0.0a0",
```

As currently spelled, this would always pull in the first nightly (*exactly* `24.12.0a0`).

Instead, we want this to be like `24.12.*,>=0.0.0a0`. That's the form that we use throughout RAPIDS ([example from cudf](https://github.com/rapidsai/cudf/blob/657f50bae866d97a231d565f34a1941efd49c721/dependencies.yaml#L614)), to limit to a single release and to allow pre-release versions to be found without needing to pass `--pre` to `pip install`.